libkbfs/folder_block_ops.go

```diff
@@ -2657,7 +2662,7 @@ func (fbo *folderBlockOps) fastForwardDirAndChildrenLocked(ctx context.Context,
 		fbo.log.CDebugf(ctx, "Fast-forwarding %v -> %v",
 			child.BlockPointer, entry.BlockPointer)
 		fbo.updatePointer(kmd, child.BlockPointer,
-			entry.BlockPointer)
+			entry.BlockPointer, false)
```

Comment:
Ooo, I forgot about the fast-forward cases.  These should also be `shouldPrefetch = true`.